appservice/src/ScmType.ts

```diff
@@ -7,5 +7,6 @@
 export enum ScmType {
     None = 'None', // default scmType
     LocalGit = 'LocalGit',
-    GitHub = 'GitHub'
+    GitHub = 'GitHub',
+    RunFromZip = 'Run-From-Zip'
```

Comment:
I don't like creating our own 'deployment type'.  I'm not convinced that Kudu will always allow us to put whatever we want here. We definitely need to discuss with the partner team because it feels like they should already define a "Run from zip" deployment type for us

Reply:
Agreed, I'm uneasy about it as well.  I'll ask them what their plans are-- I wanted to confirm whether or not a deployment was created after a run-from-zip before I commented on that thread.